src/atoms/Button/index.jsx

```diff
@@ -1,19 +1,32 @@
 // @flow
 import React from 'react';
+import cx from 'classnames';
+import './styles.scss';
 
 type Props = {
   children: React$Element<*> | string,
-  onClick: (e: SyntheticKeyboardEvent<*>) => void
+  onClick: (e: SyntheticKeyboardEvent<*>) => void,
+  className?: string
 };
 
-function Button(props: Props): React$Element<'button'> {
-  const { children, onClick } = props;
-
+function Button({
+  children,
```

Comment:
I think a `disabled` prop will be needed here.

Reply:
The button atom is not completed yet, will be included with this ticket. https://trello.com/c/qfdj7TfB/9-button-atom

In the future it would be better if we also include the trello tickets being handled on each PR.

Reply:
Ok